### ISSUE TITLE
nothing seems to be working, throwing timeout exceptions...

### ISSUE THREAD

**Issue by lextudio-support** [lextudio/sharpsnmplib#603]
I am currently running 64 bit Vista Home Premium, SP2. When I try to run the
snmpwalk.exe from the command line and specify the command like this:
snmpwalk.exe 127.0.0.1 it comes back with the following error message:

Exception of type 'Lextm.SharpSnmpLib.Messaging.TimeoutException' was thrown..
Agent: 127.0.0.1

I am using the SquareRoot (4.0) that I just downloaded. Please give me some
assistance, and  I have added snmp capabilities through windows features in
Vista.

I had previously downloaded CatPaw (5.0) but was getting the same errors, I
look to the discussions and saw that I may need to download the latest
SquareRoot instead but that didnt seem to fix my problem. These problems also
exist in the MIB browser for me, I suspect this is a problem locally.

Any advice is helpful thanks,

Steven

**Original Reported Date**: 2010-06-19T12:01:08.143-07:00
**Original CodePlex Discussion Thread ID**: 216633

**Comment by lextudio-support** [comment]
**Copied from CodePlex without authors**:

Hi Steven,

If the SNMP agent you use does not respond in a timely manner, such an
exception will be raised.

Regards,

Lex



**Original Posted Date**: 2010-06-19T17:01:01.953-07:00

**Comment by lextudio-support** [comment]
**Copied from CodePlex without authors**:

Hi Lex,



Yes I am quite aware of that...so does every snmp agent that is in the MIB
Browser also fail automatically? I am looking for suggestions as to why this
might be happening on my box. I would love to use the capabilities of #SNMP
however I am throwing this exception on every request, and everything that
comes with the SquareRoot package or CatPaw. Any advice in this direction
would be more helpful.



Thanks,

Steven



**Original Posted Date**: 2010-06-20T12:21:21.443-07:00

**Comment by lextudio-support** [comment]
**Copied from CodePlex without authors**:

Lex,

I finally found my problem, after I installed SNMP Feature for windows Vista I
was unaware that I needed to make changes to the actual service as well that
is created. I used the following link to edit the feature and the service:
<http://www.mydigitallife.info/2007/11/01/install-and-enable-snmp-service-in-
windows-xp-vista-and-2003/>



After I followed these instructions I still couldnt get snmpwalk.exe to work
on my machine, which the syntax was: snmpwalk.exe 127.0.0.1 and I was still
throwing that exception. I went back into the SNMP Service, and went to the
Security tab of the properties and instead of, "Accept SNMP packets from these
hosts", I selected, "Accept SNMP packets from any host", ran snmpwalk.exe and
worked great! This product looks like its going to be amazing!



**Original Posted Date**: 2010-06-20T18:08:55.257-07:00
